modules/cli-logging-in-web.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cli-logging-in-web_{context}"]
+= Logging in to the OpenShift CLI using the web
+
+You can log in to the OpenShift CLI (`oc`) using the web to access and manage your cluster. This allows the user to avoid inputting their password into the command line.
+// Is this accurate?
+
+== Logging in through a web browser with the CLI
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Enter the `oc login` command with the '--web' flag:
++
+[source,terminal]
+----
+$ oc login --web
+----
+
+[WARNING]
+====
+Logging in through the web browser runs a server on localhost with HTTP, not HTTPS. Use with caution on shared computers.
+====
+
+. The web browser will open automatically. If it does not, click the link in the command output.
++
+.Example output
+
+[source,terminal]
+----
+$ oc login --web <1>
+Opening login URL in the default browser: <https://openshift.example.com>
+Opening in existing browser session.
+----
+<1> If needed, you can specify the callback port by adding `--callback-port 8280 localhost:8443`
+// If I remove localhost:8443 will the command still work?
+. Enter your username and password into the corresponding browser fields. Once logged in the browser will display the text `access token received successfully; please return to your terminal`.
```

Comment:
You may need to choose an IDP first before getting to an actual login page.

Reply:
The browser isn't getting an actual access token indeed; it just prints this message to signal the successful completion of the whole process, after the callback server exchanges the authz code for a token.

https://github.com/openshift/library-go/blob/master/pkg/oauth/tokenrequest/callback_server.go#L106

Reply:
should it be this, then?

```suggestion
. Enter your username and password into the corresponding browser fields. Once logged in, the CLI will display the text `access token received successfully; please return to your terminal`.
```